frontend/src/resources/self-subject-access-review.ts

```diff
@@ -174,6 +175,22 @@ export function rbacMapping(action: string, name?: string, namespace?: string) {
                     namespace,
                 },
             ]
+        case 'cluster.destroy.detached':
+            return [
+                {
+                    resource: 'clusterdeployments',
+                    verb: 'delete',
+                    group: 'hive.openshift.io',
+                    name,
+                    namespace,
+                },
+                {
+                    resource: 'machinepools',
+                    verb: 'delete',
+                    group: 'hive.openshift.io',
+                    namespace,
+                },
+            ]
```

Comment:
@jamestalton 
Added a case to support the checks needed to destroy a detached cluster.